java-sdk/build.gradle

```diff
@@ -40,22 +40,14 @@ publishing {
                 password = System.getenv('SONATYPE_PASS')
             }
         }
-        maven {
-            name = 'artifactory'
-            url = System.getenv('ARTIFACTORY_RELEASE_URL')
-            credentials() {
-                username = System.env.ARTIFACTORY_CICD_USER
-                password = System.env.ARTIFACTORY_CICD_PASSWORD
-            }
-        }
     }
     publications {
         external(MavenPublication) {
             artifactId = rootProject.name
             from components.java
 
             artifact sourcesJar
-//            artifact javadocJar
+            artifact javadocJar
```

Comment:
fyi, i originally commented this out because it did not work, and I didn't take the time to debug